ansible/playbook/passstore/passstore_controller_inventory.yml

```diff
@@ -2,51 +2,55 @@
 # Required variables:
 #  . vm_name: Name of the vm
 #  . pass_provider: provider in the passstore project [hetzner]
-#  . k8s_type: Kubernetes host type [masters,nodes], empty for no k8s installation
-#  . k8s_version: Kubernetes version [117 ... 121], empty for no k8s installation
 - name: "Generate inventory files on the controller"
   hosts: localhost
   gather_facts: no
 
   pre_tasks:
-    # - name: "Validate required variables"
-    #   assert:
-    #     that:
-    #       - "vm_name is defined"
-    #       - "pass_provider is defined"
-    #       - "k8s_type is defined"
-    #       - "k8s_version is defined"
-    #     fail_msg: 
-    #       - "Required parameters:"
-    #       - "  vm_name: Name of the vm"
-    #       - "  pass_provider: provider in the passstore project [hetzner]"
-    #       - "  k8s_type: Kubernetes host type [masters,nodes], empty for no k8s installation"
-    #       - "  k8s_version: Kubernetes version [115,116], empty for no k8s installation"
+    - name: "Validate required variables"
+      assert:
+        that:
+          - "vm_name is defined"
+          - "pass_provider is defined"
+          - "pass_provider == 'hetzner' or pass_provider == 'openstack'"
```

Comment:
Should we keep hetzner as provider ?

Reply:
Removed the `hetzner` references and slightly improved the passwordstore documentation.